.circleci/scripts/windows_cuda_install.sh

```diff
@@ -65,6 +65,14 @@ then
     exit 1
 fi
 
+if [[ "$cuda_major_version" == "10" ]]; then
```

Comment:
Right now only 10.1 is used. So this should work. I don't if know if we have plan for 10.2, though.